runtime/src/accounts_db.rs

```diff
@@ -7103,6 +7103,11 @@ impl AccountsDb {
         config: &CalcAccountsHashConfig<'_>,
     ) -> Result<(Hash, u64), BankHashVerificationError> {
         if !use_index {
+            if !config.use_write_cache && self.accounts_cache.contains_any_slots(slot) {
+                // this indicates a race condition
+                inc_new_counter_info!("accounts_hash_items_in_write_cache", 1);
```

Comment:
Would logging slot be of any value here, maybe for correlating to other log output? I don't sift through the logs lines from accounts_db enough to know, but just throwing it out there

Reply:
I don't think so. Just the existence of the condition is sufficient.